code/03_create-excel-tables.R

```diff
@@ -0,0 +1,169 @@
+                     include_years = "all"),
+    
+    # Health Board
+    basefile %>%
+      summarise_data(category = "hb", 
+                     category_split = hb,
+                     include_years = "all"),
+    
+    # Council Area
+    basefile %>%
+      summarise_data(category = "ca", 
+                     category_split = ca,
+                     include_years = "all"),
+    
+    # HSCP
+    basefile %>%
+      summarise_data(category = "hscp", 
+                     category_split = hscp,
+                     include_years = "all"),
+    
+    # Age/Sex
+    basefile %>%
+      filter(!is.na(sex)) %>% 
+      summarise_data(category = "age/sex", 
+                     category_split = paste(age_grp, sex),
+                     include_years = "all"),
+    
+    # All Ages/Sex
+    basefile %>%
+      filter(!is.na(sex)) %>%
+      summarise_data(category = "age/sex", 
+                     category_split = paste("All Ages", sex),
+                     include_years = "all"),
+    
+    # Age/All Sex
+    basefile %>%
+      summarise_data(category = "age/sex", 
+                     category_split = paste(age_grp, "Both"),
+                     include_years = "all"),
+    
+    # All Ages/All Sex
+    basefile %>%
+      summarise_data(category = "age/sex",
+                     category_split = paste("All Ages", "Both"),
+                     include_years = "all"),
+    
+    # SIMD 
+    basefile %>%
+      summarise_data(category = "simd quintile", 
+                     category_split = simd,
+                     include_years = "all"),
+    
+    # SIMD Top 15%
+    basefile %>%
+      summarise_data(category = "simd 15", 
+                     category_split = simd_15,
+                     include_years = "all"),
+    
+    # Urban Rural 6 fold
+    basefile %>%
+      summarise_data(category = "urban rural 6", 
+                     category_split = urban_rural,
+                     include_years = "all"),
+    
+    # Urban Rural 2 fold
+    basefile %>%
+      summarise_data(category = "urban rural 2", 
+                     category_split = urban_rural_2,
+                     include_years = "all")
+    
+  )
+
+
+### 5 - Write data to excel workbooks ----
+
+figures <- loadWorkbook(here("reference-files", "figures-template.xlsm"))
+  
+writeData(figures,
+          "data",
+          excel_data,
+          startCol = 2)
+
+insertImage(figures,
+            "Figure 2",
+            here("markdown", "figures", "figure-2.png"),
+            width = 10, height = 12, 
+            units = "cm", dpi = 600,
+            startCol = 2,
+            startRow = 7)
+
+writeData(figures, 
+          "Notes", 
+          startRow = 18,
+          startCol = 3,
+          x = link)
+
+saveWorkbook(figures,
+             here("data", "output", glue("{pub_date}_figures.xlsm")),
+             overwrite = TRUE)
+
+qom <- loadWorkbook(here("reference-files", "qom-template.xlsm"))
```

Comment:
I've updated the workbook "QOM_tables_202005.xlsm" in TF folder - could you use this as the template? As I noticed in the data, health board level has "NHS " in front of the names, also has "and" for "&". So I've adjusted these things:

1. Replaced "&" with "and" in all spreadsheets.
2. Adjusted the formula to contain "NHS " for Table 1 - HB spreadsheet.